transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -9,14 +9,14 @@ transactions.remove=Removes the transaction subsystem.
+transactions.process-id-uuid=Indicates a strategy used for obtaining a unique identifier needed for the creation of transaction instances. When true, the transaction manager generates the transaction id based on the process id (PID). When set to false then the transaction manager generates the transaction id based on the referenced socket binding, i.e. the attribute 'process-id-socket-binding' is required.
 transactions.process-id-socket-binding=The name of the socket binding to use if the transaction manager should use a socket-based generation of transaction id. Will be 'undefined' if 'process-id-uuid' is 'true'; otherwise must be set.
 transactions.process-id-socket-max-ports=The maximum number of ports to search for an open port if the transaction manager should use a socket-based generation of transaction id. If the port specified by the socket binding referenced in 'process-id-socket-binding' is occupied, the next higher port will be tried until an open port is found or the number of ports specified by this attribute have been tried. Will be 'undefined' if 'process-id-uuid' is 'true'.
 transactions.jts=If true, this enables the Java Transaction Service (JTS). If JTS is enabled then the transactions attribute of the 'iiop-openjdk' subsystem must be set to 'full'. This setting changes the internal mechanisms used in the transaction manager. It has no impact on the user working with JTA API.
 transactions.use-journal-store=Use the journal store for writing transaction logs. The journal store consists of one file for all the transactions. Set to true to enable. Note that the server will not boot if both 'use-journal-store' and 'use-jdbc-store' attributes are set to true. If both are set to false then the default filesystem based store will be used.
 transactions.journal-store-enable-async-io=Whether AsyncIO should be enabled for the journal store. When true, the transaction manager will use the native aio (POSIX asynchronous I/O) libraries provided by the platform, if available. Applicable only when use-journal-store is set true.
-transactions.use-hornetq-store=Use the journal store for writing transaction logs. Set to true to enable and to false to use the default log store type. The default log store is normally one file system file per transaction log.It's alternative to jdbc based store.
+transactions.use-hornetq-store=Use the journal store for writing transaction logs. Set to true to enable and to false to use the default log store type. The default log store is normally one file system file per transaction log. It's alternative to jdbc based store.
```

Comment:
We have at least three store types so saying this could be interpreted as implying that we only have two.
Also the grammar is incorrect: `It's alternative to jdbc based store.` should be `It is an alternative to the jdbc based store` - grammarly seems to miss quite a lot so using it for this purpose is unlikely to compete with a human.

Also, JDBC is an acronym (short for Java Database Connectivity) and as such using lower case, as in `jdbc`, is syntactically incorrect. Isn't there an equivalent to grammarly for technical writing?

And has the jdbc based store been defined anywhere? Also note that we tend to use the name `JDBCStore` because that is the name we use in the Narayana documentation (it corresponds to the class name minus the package prefix). For technical documentation it is paramount that terms are well defined and used consistently. Will you check the whole document to ensure that this is the case.